game/scripts/vscripts/events.lua

```diff
@@ -145,7 +145,7 @@ function GameMode:OnPlayerLevelUp(keys)
   
   -- reduce the stat gain past a certain level
   if level > 25 then
-    local div = level / 20
+    local div = (level - 25 + 12)/12
```

Comment:
so that the `n` value is more apparent. modern compilers precalculate math using constants so it ends up executing the same exact thing, just with slightly clearer source code.